src/main/java/ua/mateacademy/tania/lesson5/taskfruits/Fruit.java

```diff
@@ -0,0 +1,51 @@
+package ua.mateacademy.tania.lesson5.taskfruits;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+
+/**
+ * Created by Tania Nebesna on 04.01.2019.
+ */
+public class Fruit {
+    private FruitKind kind;
+    private int expirationDateInDays;
+    private LocalDate deliveryDate;
+    private BigDecimal price;
+
+    public Fruit(FruitKind kind, int expirationDateInDays, LocalDate deliveryDate, BigDecimal price) {
+        this.kind = kind;
+        this.expirationDateInDays = expirationDateInDays;
+        this.deliveryDate = deliveryDate;
+        this.price = price;
+    }
+
+    public FruitKind getKind() {
+        return kind;
+    }
+
+    public int getExpirationDate() {
+        return expirationDateInDays;
+    }
+
+    public LocalDate getDeliveryDate() {
+        return deliveryDate;
+    }
+
+    public BigDecimal getPrice() {
+        return price;
+    }
+
+    public void setPrice(BigDecimal price) {
```

Comment:
You should store only immutable objects in collections. I think setters are not allowed

Reply:
Tania is true

Reply:
Not only set. Many methods of List also use hashcode and equals, and acording to this, there can be some very unexpectedly bugs. 
Anyway solution is right below)

Reply:
Ok